elements/lisk-tree/src/index.ts

```diff
@@ -18,6 +18,7 @@ import { MerkleTree } from './merkle_tree/merkle_tree';
 import { calculateMerkleRoot, calculateMerkleRootWithLeaves } from './merkle_tree/utils';
 import { SparseMerkleTree } from './sparse_merkle_tree/sparse_merkle_tree';
 import { verify, calculateRoot } from './sparse_merkle_tree/utils';
+import { Proof as SMTProof, Query as SMTQuery } from './sparse_merkle_tree/types';
```

Comment:
maybe in future we can already export SMTProof from lisk-tree as we rename it anyways

Reply:
yes, but the SMT types will be provided by lisk-db, so I will update there